.github/CODEOWNERS

```
@@ -9,3 +9,4 @@ po/it.po @detro @nicomazz @bznein
 po/ja.po @keiichiw @chikoski
 po/ko.po @jiyongp @jooyunghan
 po/pt-BR.po @rastringer @hugojacob
+po/zh-cn.po @suetfei
```

Hey @suetfei, thanks so much for signing up for this!

Based on https://github.com/google/comprehensive-rust/issues/324#issuecomment-1562327729, I propose that we call the file `zh-Hans.po` instead. I've also asked @jzpin to rename it in #676.


```suggestion
po/zh-Hans.po @suetfei
```